data-jdbc/src/test/resources/application-oracle-jsonview.yml

```diff
@@ -2,17 +2,9 @@ micronaut:
   http:
     client:
       read-timeout: 5m
-flyway:
-  datasources:
-    default:
-      locations:
-        - classpath:oracle-jsonview-migrations
-      enabled: true
-      baseline-version: 0
-      baseline-on-migrate: true
```

Comment:
Since we remove this, we should remove the `migrations/V1__Create_Schema.sql` and the flyway dependency